src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -287,7 +282,6 @@ def _get_next_sequencer(self, port, frequency, qubits: dict):
             # TODO: Throw error in that event or implement for non_overlapping_same_frequency_pulses
             # Even better, set the frequency before each pulse is played (would work with hardware modulation only)
 
-            self.ports[port].offset = qubit.sweetspot if qubit else 0
```

Comment:
This is no longer needed since the sweetspot is assigned during the platform setup().